hedera-node/test-clients/src/main/resource/bootstrap.properties

```diff
@@ -49,7 +49,7 @@ staking.startupHelper.recompute=NODE_STAKES,PENDING_REWARDS
 tokens.storeRelsOnDisk=true
 hedera.workflows.enabled=
 # Dynamic properties
-accounts.maxNumber=20_000_000
+accounts.maxNumber=20000000
```

Comment:
We can't put `_` numeric separators in this file because its the source of default values for `@HapiTest`'s; and the mod config service doesn't parse them.